src/js/content_scripts/input-mapper-utils.js

```diff
@@ -188,4 +188,156 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
             }, that.options.frequency);
         }
     };
+
+    /**
+     *
+     * Tab through the webpage using analog sticks.
+     *
+     * @param {Object} that - The inputMapper component.
+     * @param {Integer} value - The value of the gamepad input.
+     * @param {Integer} speedFactor - Times by which the tabbing speed should be increased.
+     * @param {Boolean} invert - Determine if the tabbing should be in opposite order.
+     *
+     */
+    gamepad.inputMapperUtils.analogTabbing = function (that, value, speedFactor, invert) {
+        var inversionFactor = invert ? -1 : 1;
+        value = value * inversionFactor;
+        if (value > 0.20) {
+            clearInterval(that.options.members.intervalRecords.reverseTab);
+            that.forwardTab(value, speedFactor);
+        }
+        else if (value < -0.20) {
+            clearInterval(that.options.members.intervalRecords.forwardTab);
+            that.reverseTab(-1 * value, speedFactor);
+        }
+        else {
+            clearInterval(that.options.members.intervalRecords.forwardTab);
+            clearInterval(that.options.members.intervalRecords.reverseTab);
+        }
+    };
+
+    /**
+     *
+     * Focus on the next tabbable element.
+     *
+     * @param {Object} that - The inputMapper component.
+     * @param {Integer} value - The value of the gamepad input.
+     * @param {Integer} speedFactor - Times by which the tabbing speed should be increased.
+     *
+     */
+    gamepad.inputMapperUtils.forwardTab = function (that, value, speedFactor) {
+        // Stop tabbing for the previous input value.
+        clearInterval(that.options.members.intervalRecords.forwardTab);
+
+        if (value > that.options.cutoffValue) {
+            that.options.members.intervalRecords.forwardTab = setInterval(function () {
+                // Obtain the tabbable DOM elements and sort them.
+                var tabbableElements = ally.query.tabbable({ strategy: "strict" }).sort(that.tabindexSortFilter),
+                    length = tabbableElements.length;
+
+                /**
+                 * If the body element of the page is focused or if no element is
+                 * currently focused, shift the focus to the first element. Otherwise
+                 * shift the focus to the next element.
+                 */
+                if (document.activeElement === document.querySelector("body") || !document.activeElement) {
+                    tabbableElements[0].focus();
+                }
+                else {
+                    var activeElementIndex = tabbableElements.indexOf(document.activeElement);
+
+                    /**
+                     * If the currently focused element is not found in the list, refer to
+                     * the stored value of the index.
+                     */
+                    if (activeElementIndex === -1) {
+                        activeElementIndex = that.options.members.currentTabIndex;
```

Comment:
This should be `that.currentTabIndex`, the option is only to define the member variable.